packages/cli/src/deprecation/deprecation.service.ts

```diff
@@ -47,6 +47,13 @@ export class DeprecationService {
 			message: 'MySQL and MariaDB are deprecated. Please migrate to PostgreSQL.',
 			checkValue: (value: string) => ['mysqldb', 'mariadb'].includes(value),
 		},
+		{
+			envVar: 'DB_TYPE',
+			message: 'SQLite without a pool is deprecated. Please switch to using a connection pool.',
```

Comment:
```suggestion
			message: 'Running SQLite without a pool of read connections is deprecated. Please set `DB_SQLITE_POOL_SIZE` to a value higher than zero. See: https://docs.n8n.io/hosting/configuration/environment-variables/database/#sqlite',
```

We'd need a rule of thumb for users to determine the value. This needs some research.